### PR TITLE
Edit Review

### PR DESCRIPTION
**Pull request by htran922**
- Created endpoint to get single review at `/api/v1/reviews/:id`
- Created endpoint to update a single review at `/api/v1/reviews/:id`
- Created `Update Review` form
- Made some refactors
- Added Brewfile to .gitignore

### CODE REVIEW

File: src/main/java/com/launchacademy/reviews/controllers/HomeController.java [hunk 1]
```diff
@@ -10,14 +10,16 @@
...
   })
+  
   public String forward() {
     return "forward:/";
   }
 
-    @RequestMapping(value = "/**/{path:[^\\\\.]*}")
-    public String redirect() {
-        return "forward:/404";
-    }
```
Comment (KevinTranLA) [inline src/main/java/com/launchacademy/reviews/controllers/HomeController.java:22]: This is a little weird that the code is the same but it's showing as a change. Was there an intentional spacing/tab change or was it not touched?

File: src/main/java/com/launchacademy/reviews/controllers/ReviewsApiV1Controller.java [hunk 2]
```diff
@@ -33,8 +38,41 @@ public ReviewsApiV1Controller(ReviewService reviewService, PizzaStyleService piz
...
     }
 
+    @GetMapping("/{id}")
+    public Map<String, Review> getById(@PathVariable Integer id) {
+        Map<String, Review> map = new HashMap<>();
+        Optional optional = reviewService.findById(id);
+        if (optional.isPresent()) {
+            map.put("review", (Review) optional.get());
+        } else {
+            System.out.println("Review with type with id " + id + " was not found");
```
Comment (KevinTranLA) [inline src/main/java/com/launchacademy/reviews/controllers/ReviewsApiV1Controller.java:48]: Please remember to remove once done with testing :)

**Comment by KevinTranLA** [comment]
Minor comments made, code looks good, merging!

File: src/main/java/com/launchacademy/reviews/controllers/HomeController.java (hunk 1 above)
Comment (htran922) [reply src/main/java/com/launchacademy/reviews/controllers/HomeController.java:22]: It was a spacing change after pulling from main

**Pull request merged**